Header header
int64 System
int64 SubSystem
int64 Component
int64 Diagnostic_Type
int64 Level
int64 Diagnostic_Message
string Description
